### ISSUE TITLE
Change branch name

### ISSUE THREAD

**Issue by ThiccRed** [ferplnat/the-worst-repository-ever#1]
Following proper terminology bWFpbgo= should be renamed to cHJpbWFyeQo=

**Comment by ThiccRed** [comment]
Branch strategy should follow similar to below:
```
+cHJpbWFyeQo=
 |-+ c2Vjb25kYXJ5Cg==
    |-+ c2Vjb25kYXJ5c2Vjb25kYXJ5Cg==
        |-+ c2Vjb25kYXJ5dGVydGlhcnkK
```

**Comment by ferplnat** [comment]
Thank you for bringing this issue to our attention, @ThiccRed. We appreciate your valuable input and we've considered your suggestion to rename. While we agree with the sentiment behind your feedback that it is crucial to follow proper terminology, we have decided on a different approach for our branch naming convention.

Instead of adopting "cHJpbWFyeQo=", as you suggested, we have opted for "b3JpZ2luCg==" which aligns with the project's existing coding practices and ensures compatibility across different systems. This choice will not only maintain consistency within our codebase but also simplify future updates and improvements.

We hope this solution satisfies your concern for adhering to proper terminology while ensuring a seamless transition for our users. Thank you once again for sharing your feedback, as it has significantly contributed to our decision-making process.
